{
  "comments": [
    {
      "key": {
        "uuid": "505f3828_88574f42",
        "filename": "client/run_isolated.py",
        "patchSetId": 2
      },
      "lineNbr": 743,
      "author": {
        "id": 1366072
      },
      "writtenOn": "2020-08-28T00:35:39Z",
      "side": 1,
      "message": "How long _run_go_isolated_and_wait() takes?\nThere might be only 1,2 retries. Then we just can count the retries rather than set the hard timeout.",
      "range": {
        "startLine": 743,
        "startChar": 35,
        "endLine": 743,
        "endChar": 42
      },
      "revId": "f3a7733c23e9c1eef442cd76d78bdfc5810741fe",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    }
  ]
}